src/foam/core/Validation.js

```diff
@@ -315,7 +315,7 @@ foam.CLASS({
             predicateFactory: function(e) {
               return e.REG_EXP(self, /^$|.+@.+\..+/);
             },
-            errorString: `${this.label} is not an email address`
+            errorString: 'Please enter a valid email address'
```

Comment:
"Please enter an email address in the style of person@example.com"

or just

"Please enter an email address"

Drop the work  "valid"

Reply:
went with "Please enter an email address"